src/policy/fees.cpp

```diff
@@ -672,6 +672,8 @@ CFeeRate CBlockPolicyEstimator::estimateFee(int confTarget) const
 
 CFeeRate CBlockPolicyEstimator::estimateRawFee(int confTarget, double successThreshold, FeeEstimateHorizon horizon, EstimationResult* result) const
 {
+    LOCK(m_cs_fee_estimator);
```

Comment:
`get` is a read of the ptr value, which is only updated in the constructor, so this lock is unnecessary AFAICT. `PT_GUARDED_BY` could be more appropriate.